src/core/SVG.cpp

```diff
@@ -70,10 +68,9 @@ SVG::~SVG()
     // no need to delete textures, that's handled in FBO destructor
 }
 
-void SVG::getDimensions(int &width, int &height) const
+QSize SVG::getSize() const
```

Comment:
actually doesn't work here either (error: returning reference to temporary) as svgRenderer_.defaultSize() returns a QSize